src/Evolution/VariableFixing/FixToAtmosphere.hpp

```diff
@@ -104,21 +104,32 @@ class FixToAtmosphere {
         "below TransitionDensityCutoff."};
   };
 
+  /// \brief The maximum value of the thermal specific energy
+  /// Used for 2D equations of state
```

Comment:
It might be good to document what a "reasonable" value is for different problems. I'm not sure where is the best place to do that, but here seems not terrible. Maybe:
```cpp
///
/// Reasonable values are:
/// BNS merger G=C=M_sun=1: 1.0e4
```
we can add more (e.g. disks, etc.) later. What do you think?